### PR TITLE
Em 2669 redact page

### PR DESCRIPTION
**Pull request by MichaelJacob21012**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-2669

### Change description ###
Enable redact page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/end-to-end/helpers/mvCaseHelper.js [hunk 1]
```diff
@@ -161,6 +161,16 @@ async function redactTextAndThenRemovingRedactionTest(I, caseId, mediaType) {
   await I.redactTextAndThenRemoveRedaction();
 }
 
+async function redactFirstPageTest(I, caseId, mediaType) {
+  await executeTestsOnPreview(I, caseId, mediaType);
```
Comment (phani-perla1) [inline test/end-to-end/helpers/mvCaseHelper.js:165]: We have an existing test script that redacting content on "First Page"  using below functions:
1) Draw Box function
2) Redact Text Function

So exclude this text  from master pipeline  execution and tag to "nightly build(np)"

image

**Review by phani-perla1 — CHANGES_REQUESTED** [review]

File: test/end-to-end/mvFeatures/redact.js [hunk 2]
```diff
@@ -47,6 +49,18 @@ Scenario('Redact text and then removing the redaction', async ({I}) => {
 }).tag('@ci')
   .retry(testConfig.TestRetryScenarios);
 
+Scenario('Redact first page', async ({I}) => {
+  await redactFirstPageTest(I, mvData.REDACTION_CASE, mvData.PDF_DOCUMENT);
+
+}).tag('@ci')
```
Comment (phani-perla1) [inline test/end-to-end/mvFeatures/redact.js:55]: update tag name as "@np" (nightly pipeline)

File: test/end-to-end/mvFeatures/redact.js [hunk 3]
```diff
@@ -47,6 +49,18 @@ Scenario('Redact text and then removing the redaction', async ({I}) => {
...
+  .retry(testConfig.TestRetryScenarios);
+
+Scenario('Redact multiple pages', async ({I}) => {
+  await redactMultiplePagesTest(I, mvData.REDACTION_CASE, mvData.PDF_DOCUMENT);
+
+}).tag('@ci')
```
Comment (phani-perla1) [inline test/end-to-end/mvFeatures/redact.js:61]: update tag name as "@np" (nightly pipeline)  .

This test won't run until the code is deployed on AAT  & 
moreover AAT MV execution using  different test data so you need to update the code according AAT & Preview test data.

For further details look at AAT Test Data case:
image

**Review by phani-perla1 — APPROVED** [review]
LGTM

File: test/end-to-end/mvFeatures/redact.js (hunk 2 above)
Comment (phani-perla1) [reply test/end-to-end/mvFeatures/redact.js:55]: I had a call with Michael and the new feature tests will run both AAT as well as on Preview

File: test/end-to-end/mvFeatures/redact.js (hunk 3 above)
Comment (phani-perla1) [reply test/end-to-end/mvFeatures/redact.js:61]: I had a call with Michael and the new feature tests will run both AAT as well as on Preview

File: test/end-to-end/helpers/mvCaseHelper.js (hunk 1 above)
Comment (phani-perla1) [reply test/end-to-end/helpers/mvCaseHelper.js:165]: I had a call with Michael and the new feature tests will run both AAT as well as on Preview

**Review by MunishSharmaHMCTS — APPROVED** [review]

**Pull request merged**